kv/store.go

```diff
@@ -32,6 +32,37 @@ func NewStore(client etcd.KV, jobsBucketSize uint, log *zap.SugaredLogger) *Stor
 	}
 }
 
+// CreateCheckpoint initalizes a new checkpoint for a specific workerID
```

Comment:
The comment mentions workerID but it's not used here. I think that the comment can be more generic or the argument should be called `workerID`.